internal/scorecard/scorecard.go

```diff
@@ -57,7 +59,7 @@ type pluginConfig struct {
 	External *externalPluginConfig              `mapstructure:"external,omitempty"`
 }
 
-func getPlugins() ([]Plugin, error) {
+func getPlugins(version string, selector labels.Selector) ([]Plugin, error) {
```

Comment:
Is not the selector just used [here](https://github.com/operator-framework/operator-sdk/pull/2096/files#diff-67f6fb4a2ce3f554151b70846a2f3ab3R112)? Am I right? 

So, the selector will be a const parsed, in this way, why you are passing it as arg of the function?